components/navigation.css

```diff
@@ -0,0 +1,25 @@
+.navigation {
+  width: 100%;
+  background: var(--card-bg);
+}
+
+.navigation__content {
+  width: 90%;
+  margin: 1rem auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation__logo {
+  max-width: 10rem;
+  display: flex;
+  align-items: center;
```

Comment:
Same as does not making any impact after removing it (Verified in inspect)

Reply:
removed these properties